### PR TITLE
LCD-14295 Workspace

### PR DESCRIPTION
**Pull request by petershin**
@gamerson

### CODE REVIEW

**Comment by petershin** [comment]
ci:forward

File: modules/sdk/gradle-plugins-workspace/src/main/resources/com/liferay/gradle/plugins/workspace/configurators/client_extension_Dockerfile.tpl [hunk 1]
```diff
@@ -0,0 +1,2 @@
+FROM dxp/lxc-static-resources:latest
```
[the remark this answers is not in the document]
Comment (gamerson) [reply modules/sdk/gradle-plugins-workspace/src/main/resources/com/liferay/gradle/plugins/workspace/configurators/client_extension_Dockerfile.tpl:1]: Do you want me to send a new PR ? or can you fix it?

[the remark this answers is not in the document]
Comment (petershin) [reply modules/sdk/gradle-plugins-workspace/src/main/resources/com/liferay/gradle/plugins/workspace/configurators/client_extension_Dockerfile.tpl:1]: @gamerson - please send a new pr

**Comment by petershin** [comment]
ci:stop

**Pull request closed without merging**